### PR TITLE
Update base image and optimize docker commands

### PR DESCRIPTION
**Pull request by dhafley**
Now that I found the problem with the packer version on top of an Ubuntu base, this PR will optimize docker commands and utilize a smaller base image (Alpine).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+RUN \
+  apk add --update-cache \
+    openssh-client \
+    git \
+    python \
+    py-yaml \
+    py-jinja2 \
+    py-crypto \
+    curl && \
+    rm -rf /var/cache/apk/*
```
Comment (gilday) [inline Dockerfile]: the indentation here made me think this was another package in the list

Comment (dhafley) [inline Dockerfile]: Would you suggest another RUN command?

Comment (gilday) [inline Dockerfile]: no, I would just remove two of the leading spaces on this line

Comment (gilday) [inline Dockerfile]:
```
RUN \
  apk add --update-cache \
    openssh-client \
    git \
    python \
    py-yaml \
    py-jinja2 \
    py-crypto \
    curl && \
  rm -rf /var/cache/apk/*
```

Comment (dhafley) [inline Dockerfile]: oh.  I see.  I was looking in email message where spacing was bizarre.  i'll adjust; good call.

**Pull request merged**